service/accounts/eucalyptus.py

```diff
@@ -104,21 +104,6 @@ def create_account(self, euca_user, max_quota=False, account_admin=False):
             max_quota=max_quota, account_admin=account_admin)
         return identity
 
-    def create_identity(self, euca_user, max_quota=False, account_admin=False):
```

Comment:
This is to create an identity from the eucalyptus driver as opposed to the open stack one (the function that follows that has a collision with this as it would possibly be ambiguous from the call signatures which one was meant.